backend/baton/src/test/java/touch/baton/document/alarm/delete/AlarmDeleteApiTest.java

```diff
@@ -0,0 +1,71 @@
+package touch.baton.document.alarm.delete;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import touch.baton.config.RestdocsConfig;
+import touch.baton.domain.alarm.command.Alarm;
+import touch.baton.domain.alarm.command.controller.AlarmCommandController;
+import touch.baton.domain.alarm.command.service.AlarmCommandService;
+import touch.baton.domain.member.command.Member;
+import touch.baton.fixture.domain.AlarmFixture;
+import touch.baton.fixture.domain.MemberFixture;
+
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.restdocs.headers.HeaderDocumentation.headerWithName;
+import static org.springframework.restdocs.headers.HeaderDocumentation.requestHeaders;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.delete;
+import static org.springframework.restdocs.request.RequestDocumentation.parameterWithName;
+import static org.springframework.restdocs.request.RequestDocumentation.pathParameters;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static touch.baton.fixture.vo.AlarmReferencedIdFixture.alarmReferencedId;
+
+@WebMvcTest(AlarmCommandController.class)
+class AlarmDeleteApiTest extends RestdocsConfig {
+
+    @MockBean
+    private AlarmCommandService alarmCommandService;
+
+    @BeforeEach
+    void setUp() {
+        restdocsSetUp(new AlarmCommandController(alarmCommandService));
+    }
+
+    @DisplayName("알람 삭제 API")
+    @Test
+    void deleteAlarmByAlarmId() throws Exception {
+        // given
+        final Member memberHyena = MemberFixture.createHyena();
+        final String token = getAccessTokenBySocialId(memberHyena.getSocialId().getValue());
+        final Alarm alarm = AlarmFixture.create(memberHyena, alarmReferencedId(1L));
+
+        // when
+        willDoNothing().given(alarmCommandService).deleteAlarmByMember(any(Member.class), anyLong());
```

Comment:
when 파트에 있으니깐 요건 어때용
```suggestion
    doNothing().when(alarmCommandService).deleteAlarmByMember(any(Member.class), anyLong());
```

Reply:
근데 지금까지 전체적으로 when, given 위치 지키지 않고 했어서... 이야기해보아야할 것 같아요.

Reply:
doNothing().when() 굿굿 좋습니다.